onboard-server/src/main/java/no/vegvesen/ixn/serviceprovider/OnboardRestController.java

```diff
@@ -81,7 +76,8 @@ public AddCapabilitiesResponse addCapabilities(@PathVariable String serviceProvi
 	@RequestMapping(method = RequestMethod.GET, path = "/{serviceProviderName}/capabilities", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ListCapabilitiesResponse listCapabilities(@PathVariable String serviceProviderName) {
 		OnboardMDCUtil.setLogVariables(selfService.getNodeProviderName(), serviceProviderName);
-		ServiceProvider serviceProvider = checkAndGetServiceProvider(serviceProviderName);
+		certService.checkIfCommonNameMatchesNameInApiObject(serviceProviderName);
+		ServiceProvider serviceProvider = getOrCreateServiceProvider(serviceProviderName);
```

Comment:
Should we create a new ServiceProvider when trying to list all Capabilities?

Reply:
That's the concept. We know the serviceProvider should exist, so we create it. And return an empty list from the default ServiceProvider